src/msal-client-credentials-flow/README.md

```diff
@@ -0,0 +1,163 @@
+<!-- Keeping yaml frontmatter commented out for now
+---
+# Metadata required by https://docs.microsoft.com/samples/browse/
+# Metadata properties: https://review.docs.microsoft.com/help/contribute/samples/process/onboarding?branch=main#add-metadata-to-readme
+languages:
+- csharp
+page_type: sample
+name: "ASP.NET Core minimal web API that makes a request to the Graph API as itself"
+description: "This ASP.NET Core minimal web API sample demonstrates how to issue a call to a protected API using the client credentials flow.  A request will be issued to Microsoft Graph using the application's own identity."
+products:
+- azure
+- azure-active-directory
+- ms-graph
+urlFragment: ms-identity-docs-code-app-csharp-webapi
+---
+-->
+
+<!-- SAMPLE ID: DOCS-CODE-009-->
+# ASP.NET Core minimal web API | Web API | Web API that accesses a protected web API (Microsoft Graph) | Microsoft identity platform
+
+<!-- Build badges here
+![Build passing.](https://img.shields.io/badge/build-passing-brightgreen.svg) ![Code coverage.](https://img.shields.io/badge/coverage-100%25-brightgreen.svg) ![License.](https://img.shields.io/badge/license-MIT-green.svg)
+-->
+
+This ASP.NET Core minimal web API issues a call to a protected web API (Microsoft Graph) by using the OAuth 2.0 client credentials flow. The request to the Microsoft Graph endpoint is issued using the ASP.NET Core minimal web API's own identity.
+
+```console
+$ curl https://localhost:5001/api/application
+{
+   "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#applications/$entity",
+   "id": "537a552c-58b7-4468-abdf-a7cbfa000dde",
+   "deletedDateTime": null,
+   "appId": "5b2c581d-e3ac-415e-a770-7f16254fdbf7",
```

Comment:
Have you been munging these IDs, by chance? I suspect you're deleting the apps as you go, but thought I'd check just in case.

Not strictly necessary before merge, but we'll need to add _Munge app IDs_ to the post-production list if you've not been doing so as a matter of course. (Cc: @Dickson-Mwendia)

Reply:
neither munging or deleting apps immediately but just occasionally. But yes Im planning to purge all these apps from my tenant in the near future.

Reply:
For what it's worth @mmacy, all of the GUIDs that are in the samples or screenshots that I put together are all fake.